public/app/plugins/datasource/loki/components/LokiQueryFieldForm.tsx

```diff
@@ -173,8 +185,13 @@ export class LokiQueryFieldForm extends React.PureComponent<LokiQueryFieldFormPr
               syntaxLoaded={syntaxLoaded}
             />
           </div>
+          {ExtraFieldElement ? <ExtraFieldElement {...extraFieldProps} /> : null}
```

Comment:
Is there a reason to instantiate it here? We do not seem to do any logic here so seems like having `ExtraFieldElement` already as an instantiated element and just show it as `{ExtraFieldElement}` would work.

Reply:
You're right. Changed it to ReactNode.